packages/internet-header/cypress/support/prepare-story.ts

```diff
@@ -15,14 +15,13 @@ export const prepare = (
 ) => {
   installInterceptors(config);
   cy.visitStorybook({
-    onBeforeLoad(win: { navigator: any }) {
-      // Set default browser language explicitly to English
-      Object.defineProperty(win.navigator, 'language', {
-        value: 'en',
-      });
+    qs: {
+      lang: 'de',
```

Comment:
What does this do? Is this equivalent to getting the language from the navigator?